Sources/BSON/JSON.swift

```diff
@@ -136,4 +136,33 @@ extension JSON {
     }
 }
 
+/// Helpers
+extension JSON {
+    /// Helper function used in `BSONValue` initializers that take in extended JSON and need to
+    /// check that an object has only 1 specified key.
+    ///
+    /// - Parameters:
+    ///   - key: a String representing the one key that the initializer is looking for
+    ///   - `keyPath`: an array of `String`s containing the enclosing JSON keys of the current json being passed in.
+    ///                This is used for error messages.
+    /// - Returns: JSON which is the value at the given `key` in `self`
```

Comment:
update to include what the second returned value is?

Reply:
okay, I changed it but am open to suggestions to make the wording clearer